rockcraft/providers/providers.py

```diff
@@ -87,3 +91,33 @@ def capture_logs_from_instance(instance: executor.Executor) -> None:
             emit.trace(
                 f"Could not find log file {source_log_path.as_posix()} in instance."
             )
+
+
+def get_base_configuration(
+    *, alias: bases.BuilddBaseAlias, project_name: str, project_path: Path
+):
```

Comment:
We can add a return type annotation here.

Reply:
Good catch!  Added.